components/VM_Arm/CMakeLists.txt

```diff
@@ -19,6 +19,14 @@ config_option(
     ON
 )
 
+config_option(
+    VmNoInitRam
+    VM_NO_INIT_RAM
+    "Disable init_ram module"
+    DEFAULT
+    OFF
+)
```

Comment:
Upstream workflow is complaining about the style. According to that, the option should be on one line:
`config_option(VmNoInitRam VM_NO_INIT_RAM "Disable init_ram module" DEFAULT OFF)`

Reply:
Fixed.